src/main/java/bowling/domain/BowlingGame.java

```diff
@@ -2,21 +2,41 @@
 
 public class BowlingGame {
     private final Player player;
-    private final FrameHistory frameHistory;
+    private final Records records;
+
+    public Records getRecords() {
+        return records;
+    }
+
+    public Player getPlayer() {
+        return player;
+    }
 
     public BowlingGame(Player player) {
-        this.frameHistory = new FrameHistory();
         this.player = player;
+        this.records = new Records();
     }
 
-    public void doGame() {
-        Frame frame = new Frame();
-        for(int index = 0; index < RuleConfig.NUMBER_OF_FRAME - 1; index++) {
-            frame.doFrame(frameHistory, player);
+    public boolean isEndGame() {
+        if (records.getRecordCount() >= RuleConfig.NUMBER_OF_FRAME && records.isEndLastFrame()) {
+            return true;
         }
+        return false;
```

Comment:
```suggestion
        return records.getRecordCount() >= RuleConfig.NUMBER_OF_FRAME && records.isEndLastFrame();
```

이렇게 코드를 작성해도 될 것 같아요!!